attestation-service/attestation-service/src/lib.rs

```diff
@@ -122,18 +171,29 @@ impl AttestationService {
             .await
             .map_err(|e| anyhow!("Generate reference data failed{:?}", e))?;
 
-        // Now only support using default policy to evaluate
         let evaluation_report = self
             .policy_engine
-            .evaluate(reference_data_map, tcb_json, None)
+            .evaluate(reference_data_map, tcb_json, policy_ids.clone())
             .await
             .map_err(|e| anyhow!("Policy Engine evaluation failed: {e}"))?;
 
+        let policies: Vec<_> = evaluation_report
+            .into_iter()
+            .map(|(k, v)| {
+                json!({
+                    "policy-id": k,
+                    "policy-hash": v.0,
+                    "evaluation-result": v.1,
```

Comment:
if this is always guaranteed to be `{"allow" : true }` then we don't need this field any longer

Reply:
No. as mentioned, it is determined by the rego.